docs-source/content/security/secrets.md

```diff
@@ -84,7 +106,7 @@ under **Securty**.
 #### WebLogic operator internal REST interface secret
 
 The operator exposes an internal REST HTTPS interface with a self-signed certificate.
-The certificate is kept in a Kubernetes `ConfigMap` with the name `weblogic-operator-cm ` using the key `internalOperatorCert`.
+The certificate is kept in a Kubernetes `ConfigMap` with the name `weblogic-operator-cm` using the key `internalOperatorCert`.
```

Comment:
Do customers need to directly reference this cert?  If not, I wonder if this is an internal implementation detail that could change over time - and maybe it (a) shouldn't be documented, or (b) he wording should include cautions like `the location of this secret may change without notice`.

Reply:
There are examples for making a REST call to the operator using the internal port which get this cert for the trust setup, the main example is the WLDF action scaling up the cluster.

I doubt we'd ever change the handling for internal cert and if we did then we should make the change compatible and/or document clearly.